Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1101,12 +1101,12 @@ void USpatialNetDriver::AddActorChannel(const Worker_EntityId& EntityId, USpatia
 
 USpatialActorChannel* USpatialNetDriver::GetActorChannelByEntityId(const Worker_EntityId& EntityId) const
 {
-	USpatialActorChannel* const* ActorChannel = EntityToActorChannel.Find(EntityId);
+	USpatialActorChannel* ActorChannel = EntityToActorChannel.FindRef(EntityId);
```

Comment:
Mike already merged this. (Also, I believe it just `return EntityToActorChannel.FindRef(EntityId);`)

Reply:
Yep, he mentioned it